docs/faq/storage.md

```diff
@@ -64,11 +64,15 @@ Currently, you can only achieve this manually:
 1. Copy or move files into the desired new storage location
 2. Adapt the corresponding record in the {class}`~lamindb.Storage` registry by setting the `root` field to the new location
 
-## When should I pass `key` and when should I rely purely on metadata to register a file?
+## When should I pass `key`, and when should I rely purely on metadata to register a file?
 
 The recommended way of making files findable in LaminDB is to link them to labels and use the `description` field of {attr}`~lamindb.File`.
+Generally, we discourage the usage of semantic `keys` for files due to the need for standards and potential ambiguity.
```

Comment:
Here, we could eventually link to the pros/cons that I wrote up in my other PR.

Reply:
But, given they're virtual keys now, it's also perfectly fine to have them, isn't it?

Reply:
(In the sense that they're no longer harmful.)

Reply:
It's true that they're no longer harmful with respect to storage, but they can still be abused. However, just like @Koncopd used the feature, there could be a scenario where an organization enforces `key` to be used. In such cases, I could see people not properly curating/annotating their data but rather attempting to find data by `key`. Obviously this is a bad idea because you run into the issue of people using different naming standards for `keys` and what not.

Reply:
What I'm trying to get at is that `key` can be used to model hierachies and it can be used for specific type of labels (like `ULabel`) but it is a feature that can be abused. The only sane way is not to use the feature too much.

This is what I tried to get across (and link to something else).

Do you agree and or get what I mean?

Reply:
I agree with your general point, yes! I mean, lamindb exists so that you can model data by dimensions rather than file paths.

I was just, not sure whether we should point it out in this way in the docs, because with virtual keys I can also see a folder structure and lamindb living in harmony.

But you're right that it could be abused and people don't use registries but just file paths. On the other hand, why would they use lamindb then? 🤔

Reply:
There is a whole can of worms here around data governance that will give us a lot of headache. I'm not sure whether we'll actually be able to ever enforce policies that encourage well-annotated validated data, but I'm still hopeful that we can solve this on the API-level.

The first (and maybe only) example is that you'll get a warning if you want to save a non-consecutively executed notebook. And that warning will translate in a warning badge on the hub. 

So, I can see something similar for files under certain conditions. But in other conditions (e.g. a BFX pipeline output), people might really just want to store things by key and not bother about labeling beyond the automated provenance.